iphone/Maps/Core/Location/MWMLocationManager.mm

```diff
@@ -34,6 +34,49 @@ typedef NS_OPTIONS(NSUInteger, MWMLocationFrameworkUpdate) {
   BicycleRouting
 };
 
+static std::string DebugPrint(GeoMode geoMode) {
```

Comment:
static global functions in .cpp/.mm files are not visible to other code.
anonymous functions without a static keyword in the anonymous namespace (like here) are also not visible to other code.

So if `namespace {}` is used, then `static` keyword is redundant. It is needed only if the function is in the global namespace, not in the anonymous namespace.

Reply:
Thanks!